tools/test-harnesses/src/utils/checkContainerType/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copied from [dom-testing-library](https://github.com/testing-library/dom-testing-library/blob/bd04cf95a1ed85a2238f7dfc1a77d5d16b4f59dc/src/helpers.ts#L65)
```

Comment:
do we still need this?

Reply:
Since I straight up copied this, It felt right to mark it as copied, but if you think it's not necessary, I can remove it.

Reply:
I mean, since we're installing `@testing-library/dom`, do we need this file?

Reply:
oh i see, this function isn't exported from `@testing-library/dom`

Reply:
yess